GraphQL-TSC.md

```diff
@@ -0,0 +1,253 @@
+
+# GraphQL Technical Steering Committee (TSC)
+
+The GraphQL TSC will be responsible for all technical oversight of the open source project. In particular, the TSC is responsible for approving official GraphQL specification releases and coordinating between projects.
```

Comment:
I guess plural "projects" might be more accurate.

Reply:
Good point, that does look better. I just pushed an update.